tests/cases/trait/method-defaults

```diff
@@ -11,11 +11,11 @@ namespace Vendor;
 
 trait Foo
 {
-    public function bar($baz = [], $qux) : void
+    public function bar(string $baz = '', string $qux) : void
```

Comment:
Was the change from array to string intentional?

Reply:
Was copied from #32, was just to reduce the amount of changes (as now need to say array of _what_). But that has been done in some places, so will probably tweak to be consistent (not sure which though).

Reply:
cool, so not a bug in the auto-fixer! Makes sense though